packages/gestalt-design-tokens/build.js

```diff
@@ -190,6 +192,25 @@ const filterList = [
   filterFontWeight,
 ];
 
+const getPrefix = (filePath) => {
+  const filePaths = filePath.split('/');
+  const filename = path.parse(filePath).name;
+
+  if (filePaths.includes('sema') || filename.startsWith('sema-')) {
```

Comment:
Does this mean `sema` token files can exist outside of `sema` folder?

Reply:
Yes, they can either be prefixed, or sema can be in the filepath. 

We have `classic` and `vr-theme` both of which are organized differently